Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -92,6 +109,32 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
         self.present(alertController, animated: true, completion: nil)
     }
     
+    // MARK: - Private instance methods
+
+    private func toggleView() {
+        let syncsCount: Int? = Settings.readOptionalValue(for: Settings.Keys.SyncsCount)
+        if isNewWalletSetup && !isSynced {
+            self.showFirstWalletAddressAndQRCode()
+            self.populateWalletDropdownMenu()
+            contentStackView.isHidden = false
+            syncInProgressLabel.isHidden = true
+        } else if !isSynced && syncsCount != 1 {
+            contentStackView.isHidden = true
+            syncInProgressLabel.isHidden = false
+        } else if isSynced {
+            self.showFirstWalletAddressAndQRCode()
+            self.populateWalletDropdownMenu()
+            contentStackView.isHidden = !contentStackView.isHidden
+            syncInProgressLabel.isHidden = isSynced
+        }
+    }
+
+    private func setUpConstraints() {
```

Comment:
Rename to `setupSyncInProgressLabelConstraints`. Maybe add a brief comment in the method body explaining how/where the codes below position the label so we don't always need to run the math on our heads when reading this section of this code.